modules/oadp-configuring-dpa-multiple-bsl.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-dpa-multiple-bsl_{context}"]
+= Configuring the DPA with more than one BSL
+
+You configure the DPA with more than one BSL and specify the credentials provided by the cloud provider.  
```

Comment:
you can?

Reply:
Is a "can" missing between You and configure at the beginning of the sentence?

Reply:
done.